pyenvbuilder/utils.py

```diff
@@ -0,0 +1,81 @@
+import logging
+import subprocess
+from pathlib import Path
+
+
+logger = logging.getLogger(__name__)
+
+
+def validate_installed(args):
+    '''
+    Invokes the args and returns True if successful
+    '''
+    try:
+        if subprocess.run(args, check=True, stdout=subprocess.DEVNULL):
+            return True, f''
+    except subprocess.CalledProcessError as err:
+        logger.error(err)
+        return False, err
+    except FileNotFoundError:
+        msg = f'{args} not found, please install {args} before proceeding.'
+        logger.error(msg)
+        return False, msg
+
+
+def validate_path(args):
```

Comment:
Same as above

Reply:
got this one too

Reply:
Usually `args` means a list or arguments... Here you are accepting just one file, right? Maybe just arg or some other name?